base_custom_info/models/custom_info_value.py

```diff
@@ -225,12 +226,6 @@ def _onchange_property_set_default_value(self):
             if not record.field_type and record.property_id.field_type:
                 record.field_type = record.property_id.field_type
 
-    @api.onchange("value")
-    def _onchange_value(self):
```

Comment:
This is still needed anyway. Please don't remove it.

Reply:
Sure. But would you care to explain? I'd think it would get triggered with inverse function in place.

Reply:
Just read the docstring for the explanation.

Reply:
@pedrobaeza done.